src/rustDemangle.cpp

```diff
@@ -45,10 +45,10 @@ struct demangle_legacy {
 
 // private version of memrchr to avoid _GNU_SOURCE
 static void *demangle_memrchr(const void *s, int c, size_t n) {
-    const uint8_t *s_ = (uint8_t *)s;
+    const uint8_t *bytes = (uint8_t *)s;
```

Comment:
This file came from another project. I don't want it to diverge from the original source.

Reply:
Fixed in 5e3d572737fdb2cac2498349010e3949fd1c1962